### PR TITLE
Prevent `start_for_complement.sh` from setting `START_POSTGRES` to `false` when it's already set

### PR DESCRIPTION
**Pull request by anoadragon453**
I have a use case where I'd like the Synapse image to start up a postgres instance that I can use, but don't want to force Synapse to use postgres as well.

This commit prevents postgres from being started when it has already been explicitly enabled elsewhere.

### CODE REVIEW

File: docker/complement/conf/start_for_complement.sh [hunk 1]
```diff
@@ -32,8 +32,13 @@ case "$SYNAPSE_COMPLEMENT_DATABASE" in
     ;;
 
   sqlite|"")
-    # Configure supervisord not to start Postgres, as we don't need it
-    export START_POSTGRES=false
```
Comment (reivilibre) [inline docker/complement/conf/start_for_complement.sh:36]: would be tempted to do `export START_POSTGRES=${START_POSTGRES:-false}`, or at least I find that easier to read

**Review by reivilibre — APPROVED** [review]

File: docker/complement/conf/start_for_complement.sh (hunk 1 above)
Comment (anoadragon453) [reply docker/complement/conf/start_for_complement.sh:36]: Great suggestion!

File: changelog.d/16985.misc [hunk 2]
```diff
@@ -0,0 +1 @@
+Allow containers building on top of Synapse's Complement container is use the included PostgreSQL cluster.
```
Comment (MadLittleMods) [inline changelog.d/16985.misc:1]: > I have a use case where I'd like the Synapse image to start up a postgres instance that I can use, but don't want to force Synapse to use postgres as well.

What was the use case?

Comment (MadLittleMods) [reply changelog.d/16985.misc:1]: -> https://github.com/element-hq/ti-messenger-proxy/pull/137

**Pull request merged**